pkg/virt-launcher/virtwrap/live-migration-target.go

```diff
@@ -30,70 +27,16 @@ func (l *LibvirtDomainManager) finalizeMigrationTarget(vmi *v1.VirtualMachineIns
 }
 
 func (l *LibvirtDomainManager) prepareMigrationTarget(vmi *v1.VirtualMachineInstance, useEmulation bool) error {
+	domain := &api.Domain{}
 
 	logger := log.Log.Object(vmi)
 
-	var emulatorThreadCpu *int
-	domain := &api.Domain{}
-	podCPUSet, err := util.GetPodCPUSet()
+	c, err := l.generateConverterContext(vmi, useEmulation, nil, true)
 	if err != nil {
-		logger.Reason(err).Error("failed to read pod cpuset.")
-		return fmt.Errorf("failed to read pod cpuset: %v", err)
-	}
-	// reserve the last cpu for the emulator thread
-	if vmi.IsCPUDedicated() && vmi.Spec.Domain.CPU.IsolateEmulatorThread {
-		if len(podCPUSet) > 0 {
-			emulatorThreadCpu = &podCPUSet[len(podCPUSet)]
-			podCPUSet = podCPUSet[:len(podCPUSet)-1]
-		}
+		logger.Reason(err).Error("Failed to generate libvirt domain from VMI spec")
```

Comment:
This seems redundant as it is already logged at `SyncMigrationTarget`.

Reply:
fixed